helm/ph-ee-engine/integration-test/templates/myjob.yaml

```diff
@@ -0,0 +1,76 @@
+apiVersion: batch/v1
```

Comment:
can you tell me how can you enable/diable this job?
also how to execute this job during helm upgrade?